src/main/resources/ValidationMessages.properties

```diff
@@ -1,7 +1,11 @@
 #customise standard errors
 javax.validation.constraints.NotNull.message=mandatory_element_missing
+javax.validation.constraints.NotBlank.message=mandatory_element_missing	
```

Comment:
There's a tab following this property which shouldn't be there and is causing issues when integrating with the web